packages/stream_chat_flutter/lib/src/channel/stream_message_preview_text.dart

```diff
@@ -20,6 +20,17 @@ class StreamMessagePreviewText extends StatelessWidget {
   /// The style to use for the text.
   final TextStyle? textStyle;
 
+  /// The default preview predicate which determines if the message should be
+  /// shown in the preview or not.
+  static bool defaultPredicate(Message message) {
```

Comment:
Is `defaultPredicate` also something we use on the other sdk's? It's not a word that I immediately understand what it's doing.

Reply:
Ios have it named `previewMessagePredicate` but i named it just 	`defaultPredicate` as it can only get called from the `StreamMessagePreviewText`. So basically

```dart
StreamMessagePreviewText.defaultPredicate
```